src/test/java/bio/terra/integration/FileTest.java

```diff
@@ -46,7 +46,7 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
-@ActiveProfiles({"google", "integrationtest"})
+@ActiveProfiles({"google", "integrationtest", "alpha"})
```

Comment:
Same with this - we can set the active profiles in alpha helm, right?

Reply:
We set `SPRING_PROFILES_ACTIVE` to `alpha` in Helm, but the special `@ActiveProfiles` annotation ignores anything from environment variables and `SPRING_PROFILES_ACTIVE`, so we still need it for the integration tests.